Ilya/NetflixRoulette/netflix-roulette/src/shared/pagination/Pagination.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import "./Pagination.scss";
+import { RouteComponentProps } from "react-router-dom";
+import { HomePageConnectProps } from 'pages/home';
+import queryString from "query-string";
+
+type IPaginationProps = HomePageConnectProps & RouteComponentProps;
+
+class Pagination extends React.Component<IPaginationProps> {
```

Comment:
It's good that you've made a pagination 👍. But I see that you put requests inside of it. Pagination is a reusable component, so imagine that we gonna add one more page with, for example, users favourite movies and request will be different, so we will need to rewrite your pagination, add a conditional or smth. So it's not a good approach to do some specific to the page logic inside of common component. You have to pass onChangePage, onMoveToLimitPage props instead.

Reply:
Done